core/modules/field_ui/field_ui.admin.inc

```diff
@@ -1174,6 +1211,64 @@ function field_ui_display_overview_form($form, &$form_state, $entity_type, $bund
   return $form;
 }
 
+/**
+ * Form submission handler for field_ui_display_overview_form().
```

Comment:
Docblock should say `field_ui_display_form()`, not `field_ui_display_overview_form()`